src/amo/actions/landing.js

```diff
@@ -0,0 +1,29 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export function getLanding({ addonType }) {
+  return {
+    type: LANDING_GET,
+    payload: { addonType },
+  };
+}
+
+export function loadLanding(
+  { addonType, entities, featured, highlyRated, popular }
```

Comment:
I'd like to land #1461 but I think it's overkill for this particular method, especially as we don't tend to do it elsewhere IIRC.